engines/wintermute/base/gfx/opengl/base_render_opengl3d.cpp

```diff
@@ -383,6 +383,30 @@ bool BaseRenderOpenGL3D::setup3D(Camera3D *camera, bool force) {
 			glGetFloatv(GL_MODELVIEW_MATRIX, _lastViewMatrix.getData());
 		}
 
+		bool fogEnabled;
+		uint32 fogColor;
+		float fogStart;
+		float fogEnd;
+
+		_gameRef->getFogParams(&fogEnabled, &fogColor, &fogStart, &fogEnd);
+
+		if (fogEnabled) {
+			glEnable(GL_FOG);
+			glFogi(GL_FOG_MODE, GL_LINEAR);
+			glFogf(GL_FOG_START, fogStart);
+			glFogf(GL_FOG_END, fogEnd);
+
+			GLfloat color[4];
```

Comment:
Just assign the below directly in the initializer.

Reply:
Done.